### PR TITLE
Help & todo

### PR DESCRIPTION
**Pull request by matous-volf**

### CODE REVIEW

**Comment by matous-volf** [comment]
Screenshot from 2023-06-22 20-35-29

File: webapp/templates/public/help/registration.html.twig [hunk 1]
```diff
@@ -15,7 +15,6 @@
     <img src="{{ asset('images/help/register.png') }}" alt="Registrace" class="mb-3 rounded-3 img-fluid"
          style="max-height: 400px">
     <p>
-        Následně se svými údaji <a href="/login">přihlaste</a>. V případě, že zapomenete heslo, napište nám na
-        <a href="mailto:example@delta-skola.cz">example@delta-skola.cz</a> a my vám ho resetujeme.
+        Následně se svými údaji <a href="/login">přihlaste</a>.
```
Comment (zapotocnylubos) [inline webapp/templates/public/help/registration.html.twig]: umime tady generovat dynamicky url podle route name?

Comment (matous-volf) [reply webapp/templates/public/help/registration.html.twig]: jak to myslis? jak by se url mela menit?

Comment (matous-volf) [reply webapp/templates/public/help/registration.html.twig]: myslis takhle (viz commit)?
to funguje

**Review by zapotocnylubos — APPROVED** [review]

**Pull request merged**